ui-components/src/notifications/ApplicationStatus.tsx

```diff
@@ -1,62 +1,49 @@
 import * as React from "react"
 import moment from "moment"
 import { t } from "../helpers/translator"
-import { Listing } from "@bloom-housing/backend-core/types"
-import { openDateState } from "../helpers/state"
 import { Icon, IconFillColors } from "../icons/Icon"
+import { ApplicationStatusType } from "../global/ApplicationStatusType"
 import "./ApplicationStatus.scss"
 
 export interface ApplicationStatusProps {
-  listing: Listing
+  content: string
+  status: ApplicationStatusType
+  date?: Date
+  showTime?: boolean
```

Comment:
@seanmalbert i'm wondering now if date should just be omitted as a property and instead should just already be formatted and included in `content`.  i can see why components were passing `Listing`s around because otherwise, there'd be props being passed to "parent" components so that they in turn could be passed to "child" components.

As an example, the [ImageCard](https://github.com/bloom-housing/bloom/pull/1555/files#diff-d38145120a0c1e6785134db0b5ddde8def3eb1803d77e72c6efce9371048c118R29) component attempting to use the ApplicationStatus component.  The way it's been refactored now, the ImageCard props would need to accept a date prop in order to pass it along to the ApplicationStatus component.

Reply:
Yes, per your comment in ApplicationStatus, I think it makes sense that the date should already be formatted and part of the content.